server/server.go

```diff
@@ -1209,20 +1209,20 @@ func (s *Server) ReplicateFileToAllMembers(ctx context.Context, name string, dat
 	for _, member := range resp.Members {
 		clientUrls := member.GetClientUrls()
 		if len(clientUrls) == 0 {
-			log.Warn("failed to replicate file", zap.String("name", name), zap.String("member", member.GetName()), errs.ZapError(err))
-			return errors.Errorf("failed to replicate to member %s: clientUrls is empty", member.GetName())
+			log.Warn("failed to replicate file", zap.String("name", name), zap.String("member", member.GetName()), zap.Error(err))
```

Comment:
Should we use `errs.ZapError`? cc @rleungx

Reply:
Yes